mscore/editstyle.ui

```diff
@@ -6,7 +6,7 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>923</width>
+    <width>1150</width>
```

Comment:
Artifact.  I believe we override this in the code anyhow, to get rid of the constant issues we were having with this dialog?